packages/components/pie-card-container/src/defs.ts

```diff
@@ -47,4 +49,14 @@ export interface CardContainerProps {
      * What the interaction intent of the card should be such as anchor, button or regular div (default).
      */
      interactionType?: typeof interactionTypes[number];
+
+    /**
+     * An array of padding values from a to g.
+     */
+    padding?: PaddingValue | PaddingValue[];
+
+    /**
+     * An array of padding values from a to g.
+     */
+    paddingX?: PaddingValue | PaddingValue[];
```

Comment:
What's the difference between `padding` and `paddingX`?

Reply:
Just wondering as well – could this be handled with a single padding prop? (similar to how CSS does with `padding: a b c d;`?

Reply:
So i did initially try that but the issue is that our valid prop function (the thing used to make sure we pass in valid values) wont allow a mixture of different values so i split it out (similar to what snacks does). The `padding` one is for a single value which can go all the way around the component i.e `padding: 5px` and the `paddingX` is one that if supplied will apply to left and right so if both are used together then it would output like `padding: 5px 10px` 10px being the paddingX value. If only `paddingX` is provided and not `padding` then no padding is rendered.

Reply:
Fixed.